common/app-starters-security-common/README.adoc

```diff
@@ -1,10 +1,21 @@
 === `App Starters Security` Common Module
 
-This artifact contains a Spring Boot auto-configuration to disable security in the application that set `spring.cloud.stream.security.enabled = false` (default).
-If the `spring.cloud.stream.security.enabled` property is explicitly set to `true` then this configuration falls back to the default Spring Security configuration.
+Spring Boot auto-configuration to manage the web security of the application starters.
+
+When the `app-starters-security-common` dependency is on the classpath, the `spring.cloud.streamapp.security.enabled` and `spring.cloud.streamapp.security.csrf-enabled` properties control the application security behavior.
+
+By default the security is enabled and only
```

Comment:
Looks like you haven't finished the sentence or would like to connect this one with the next...

Reply:
ah, just fixed